src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -375,31 +360,31 @@ public void setStatefulSet(
 	 */
 	@Deprecated
```

Comment:
add `@deprecated Use {@link #getLimits()}` much like the one in `getCpu`?

Reply:
yes its there already